### PR TITLE
Equation 1 and 2 of lcm inequality lemma

### PR DESCRIPTION
**Pull request by metakunt**
See https://www3.nd.edu/~andyp/notes/AKS.pdf
For the next part I will need itgpowd, so I've moved it to main. 

Also I've changed cniccibl to its choice free version. I am wondering if we need countable choice to define integrals.

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -137786,6 +137786,18 @@ the existence of its supremum (see ~ suprcl ).  (Contributed by Paul
   elicc01 $p |- ( X e. ( 0 [,] 1 ) <-> ( X e. RR /\ 0 <_ X /\ X <_ 1 ) ) $=
     ( cc0 c1 0re 1re elicc2i ) BCADEF $.
 
+  $( The closed unit interval is a subset of the set of the real numbers.
+     Useful lemma for manipulating probabilities within the closed unit
+     interval.  (Contributed by Thierry Arnoux, 21-Dec-2016.) $)
+  elunitrn $p |- ( A e. ( 0 [,] 1 ) -> A e. RR ) $=
+    ( cc0 c1 cicc co wcel cr cle wbr elicc01 simp1bi ) ABCDEFAGFBAHIACHIAJK $.
+
+  $( The closed unit interval is a subset of the set of the complext numbers.
```
Comment (wlammen) [inline set.mm:137795]: typo: complext

File: set.mm [hunk 2]
```diff
@@ -633173,10 +633227,91 @@ fixed reference functional determined by this vector (corresponding to
...
+
+  ${
+    $d M k x $.  $d N k x $.  $d k ph x $.
+    lcmineqlem2.1 $e |- F = S. ( 0 [,] 1 ) ( ( x ^ ( M - 1 ) ) x.
+     ( ( 1 - x ) ^ ( N - M ) ) ) _d x $.
+    lcmineqlem2.2 $e |- ( ph -> N e. NN ) $.
+    lcmineqlem2.3 $e |- ( ph -> M e. NN ) $.
+    lcmineqlem2.4 $e |- ( ph -> M <_ N ) $.
+    $( Part of lcm inequality lemma, this part eventually shows that F times
+       the least common multiple of 1 to n is a integer.  (Contributed by
```
Comment (wlammen) [inline set.mm:633288]: typo: is an integer

**Review by wlammen — CHANGES_REQUESTED** [review]

**Comment by avekens** [comment]
> For the next part I will need itgpowd, so I've moved it to main.

Theorems should not be moved to main unless they are actually used. We cannot be sure if they will be used in the (near) future, so such a moving cannot be approved.

**Comment by metakunt** [comment]
Alright, I guess I'll move this to draft until I can prove lcmineqlem3

**Review by wlammen — DISMISSED** [review]

**Comment by metakunt** [comment]
I finished proving lcmineqlem3 @avekens As expected it uses itgpowd

**Review by avekens — APPROVED** [review]

**Review by tirix — APPROVED** [review]

**Comment by tirix** [comment]
FYI, the proof with "math display" is out here: [https://metamath.tirix.org/mpests/lcmineqlem3](https://metamath.tirix.org/mpests/lcmineqlem3)

**Pull request merged**